src/humancellatlas/data/metadata/__init__.py

```diff
@@ -576,10 +576,9 @@ def specimens(self) -> List[SpecimenFromOrganism]:
         return [s for s in self.biomaterials.values() if isinstance(s, SpecimenFromOrganism)]
 
     @property
-    def sequencing_input(self) -> List[CellSuspension]:
+    def sequencing_input(self) -> List[Biomaterial]:
         return [bm for bm in self.biomaterials.values()
-                if isinstance(bm, CellSuspension)
-                and any(ps.is_sequencing_process() for ps in bm.to_processes.values())]
+                if any(ps.is_sequencing_process() for ps in bm.to_processes.values())]
```

Comment:
For a complete validation, I compared the results of our old approach with this approach on bundle `fc17a456-6c9b-4e62-a095-4b00c2e042c4.2018-08-23T133206.222169Z` in integration DSS:

```python
sequencing_protocol = [plv for plk, plv in bundle.protocols.items() if isinstance(plv, SequencingProtocol)]
sequencing_protocol_id = sequencing_protocol[0].document_id
sequencing_protocols = [link for link in bundle.links if link.source_type == 'process' and link.destination_id == sequencing_protocol_id]
process_id = sequencing_protocols[0].source_id
sample_links = [link for link in bundle.links if link.source_type == 'biomaterial' and link.destination_id == process_id]
sequencing_input_id = sample_links[0].source_id
```
Our old approach gives `880889e6-5399-4af8-9e84-e4bf4c130b63` in a fussy way like it showed above, which matches the `SpecimenFromOrganism .document_id` returned by this approach, so I'm more confident now.